crates/bevy_pbr/src/render/light.rs

```diff
@@ -518,6 +522,7 @@ pub fn extract_lights(
                     frusta: extracted_frusta,
                     render_layers: maybe_layers.unwrap_or_default().clone(),
                     occlusion_culling,
+                    angular_size: sun_light.map_or(0.0, |sun_light| sun_light.angular_size),
```

Comment:
shouldnt this default to the default angular size?
```suggestion
                    angular_size: sun_light.unwrap_or_default().angular_size,
```

Reply:
Good suggestion! for this to work /compile I had  to add
```
impl Default for &SunLight {
    fn default() -> Self {
        &SunLight::SUN
    }
}
```

Reply:
ah, i thought that was already there